compaction/compaction_manager.cc

```diff
@@ -718,6 +718,8 @@ future<> compaction_manager::rewrite_sstables(column_family* cf, sstables::compa
             };
 
             return with_semaphore(_rewrite_sstables_sem, 1, [this, task, &cf, descriptor = std::move(descriptor)] () mutable {
+              // Take write lock for cf to serialize cleanup/upgrade sstables/scrub with major compaction/reshape/reshard.
+              return with_lock(_compaction_locks[&cf].for_write(), [this, task, &cf, descriptor = std::move(descriptor)] () mutable {
```

Comment:
nitpick: github is good at presenting a diff even if indentation changes - so perhaps just use correct indentation here from the start instead of this half-indent?

Reply:
fixed in v4

Reply:
is it? (: I still see a 2-space indent here. I'll check it out via git just to be double-sure

Reply:
@bhalevy aaah I see - I didn't mean to squash the reindentation commit, just to use a full 4-space indentation in here :)

Reply:
Best to unindent via coroutines